src/Components/WebAssembly/WebAssembly/src/Services/LazyAssemblyLoader.cs

```diff
@@ -18,8 +18,6 @@ namespace Microsoft.AspNetCore.Components.WebAssembly.Services;
 /// </summary>
 public sealed partial class LazyAssemblyLoader
 {
-    private HashSet<string>? _loadedAssemblyCache;
```

Comment:
This was probably preventing duplicate load, is that still working ?

Reply:
It is moved to runtime for WebAssembly https://github.com/dotnet/runtime/blob/934e7e475e3ce71539d2065ae3e06d19a664b46f/src/mono/wasm/runtime/lazyLoading.ts#L20